app/src/main/java/io/intrepid/contest/models/Contest.java

```diff
@@ -0,0 +1,99 @@
+package io.intrepid.contest.models;
+
+import android.os.Parcel;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import timber.log.Timber;
+
+public class Contest {
+    private UUID contestId;
+    private UUID creatorId;
+    private Date creationDate;
+    private Date lastUpdatedDate;
+    private String title;
+    private String description;
+    private List<Category> categories;
+
+    @Override
+    public String toString() {
+        return title + "/n" +
+                description + "/n" +
+                contestId + "/n" +
+                creatorId + " " + creationDate +
+                " /n size of categories was " + categories.size() +
+                "categories " + categories +
+                " /n Last updated " + lastUpdatedDate;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public static class Builder {
+        public static final int UUID_MIN_LIMIT = 0;
+        public static final int UUID_MAX_LIMIT = Integer.MAX_VALUE;
+        public List<Category> categories = new ArrayList<>();
+        public UUID contestId;
+        public UUID creatorId;
+        public Date creationDate;
+        public Date lastUpdatedDate;
+        public String title;
+        public String description;
+
+        public Builder() {
+            this(new UUID(UUID_MIN_LIMIT, UUID_MAX_LIMIT));
+        }
+
+        public Builder(UUID creatorId) {  //fixme
```

Comment:
True the UUID will be received though the retrofitting so i'm not quite there yet. I'll remove this for now